pages/tag/_query/index.vue

```diff
@@ -1,11 +1,11 @@
 <template>
-	<b-row align-h="center">
+	<div align-h="center">
```

Comment:
Without bootstrap-vue, `align-h` doesn't do anything. Please remove it throughout the components.